INSTALL.md

```diff
@@ -73,6 +73,46 @@ Configuration for Softwerkskammer and SoCraTes:
 * Copy the logging configuration file `config-examples/winston-config.json` to `config/winston-config.json`, and adapt the paths if you like.
 * Copy the mailsender configuration file `config-examples/mailsender-config.json` to `config/mailsender-config.json`. Without setting up a proper server sending mails won't work but this configuration is sufficient to be able to start both the softwerkskammer and socrates app.
 
+#### Mailserver settings
+if you want to be able to send mails, you need to configure the mail sender. One way to achieve a running configuration
+is to use a mail server, you have access to. The configuration should look like:
+
+###### config/mailsender-config.json
+
+<pre><code>
```

Comment:
because I didn't want to change an example that runs and can be used without changes if you don't need the email functionality.